src/D3MHub.sol

```diff
@@ -416,7 +368,9 @@ contract D3MHub {
                 currentAssets
             );
         } else if (ilks[ilk].tic != 0 || !ilks[ilk].plan.active()) {
-            _fix(ilk, address(_pool));
+            if (ilks[ilk].culled == 0) {
```

Comment:
`lks[ilk].culled` can be read to a local variable here instead of being read twice from storage.

Reply:
I've re-structured this further so this should be solved now.